### ISSUE TITLE
Cannot install transitive dependencies

### ISSUE THREAD

**Issue by nanafan93** [intive-FDV/DynamicJasper#149]
I am trying to install the latest version of DynamicJasper using gradle and keep getting an where it can't find a transitive dependency. 

`Could not find com.github.librepdf:openpdf:1.3.30.jaspersoft.2.`
It tells me that the dependency is required by DJ -> Jasper -> like so 
`Required by XXX > ar.com.fdvs:DynamicJasper:5.3.9 > net.sf.jasperreports:jasperreports:6.20.4`

Do I need to look for it in a specific repository ?

**Comment by danielpeintner** [comment]
I don't know your setup but you might want to add `net.sf.jasperreports:jasperreports:6.21.3` as dependency in your POM/gradle which resolves fine in my case.

**Comment by nanafan93** [comment]
Yeah it worked fine with `6.20.3` as well but here's the problem. 

I was looking at the `pom.xml` in this project and it specifies the following. 

```
        <dependency>
            <groupId>net.sf.jasperreports</groupId>
            <artifactId>jasperreports</artifactId>
            <version>[6.20.1, 6.20.4]</version>
            <exclusions>
                <exclusion>
                    <groupId>commons-collections</groupId>
                    <artifactId>commons-collections</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>commons-logging</groupId>
                    <artifactId>commons-logging</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>bouncycastle</groupId>
                    <artifactId>bcmail-jdk14</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>bouncycastle</groupId>
                    <artifactId>bcprov-jdk14</artifactId>
                </exclusion>
            </exclusions>
        </dependency>

```
Gradle by default tries to get the latest version i.e 6.20.4 but fails to do so. I wonder if this is worth addressing since it is not working out of the box. I wonder if it even can be addressed here since its failing to pull a dependency which has nothing to do with this project really... Just want to get an idea of what action is recommended in such a case

**Comment by danielpeintner** [comment]
2 points:
1. Jasperreport changed the dependency from iText to librepdf some months ago. Maybe that is causing problems
2. DynamicJasper is missing recent activity, and I don't see anyone working on it. It is a pity but unless someone picks up the work, I don't think anything will change in the near future (I hope I am wrong).
